css/style.css

```diff
@@ -30,7 +41,172 @@
     float: right;
     margin-top: 50px;
     border: 1px solid grey;
-    padding: 10px;
+    padding: 10px 15px;
+    font-size: 17px;
     background: #2196F3;
     color: white;
+}
+
+
+/*Slider one*/
+
+.site-slider {
+    position: relative;
+}
+
+.position-top {
+    position: absolute;
+    top: 50%;
+}
+
+.site-slider .slider-btn .prev,
+.site-slider .slider-btn .next {
+    background-color: #6983aa;
+    padding: 10px 20px;
+    border-radius: 100%;
+    color: white;
+    margin: 0 10px;
+    opacity: 0;
+    transition: opacity 1s ease;
+}
+
+.site-slider:hover .slider-btn .prev,
+.site-slider:hover .slider-btn .next {
+    opacity: 1;
+}
+
+.right-0 {
+    right: 0;
+}
+
+
+/*SLick Dots*/
+
+.slick-dots {
+    display: flex;
+    justify-content: center;
+    margin: 0;
+    padding: 10px 0;
+    list-style-type: none;
+    position: absolute;
+    bottom: 0;
+    right: 50%;
+}
+
+.slick-dots li {
+    margin: 0 10px;
+}
+
+.slick-dots button {
+    width: 10px;
+    height: 10px;
+    padding: 0;
+    border: none;
+    text-indent: -99999px;
+    border-radius: 100%;
+}
+
+.slick-dots li.slick-active button {
+    transform: scale(2.5);
+    box-shadow: 0px 3px 5px 1px rgba(0, 0, 0, 0.205);
+}
+
+
+/*Slider two*/
+
+.site-slider-two {
+    position: relative;
+}
+
+.site-slider-two .product {
+    height: 100%;
+    position: relative;
+    overflow: hidden;
+}
+
+.site-slider-two .product img {
+    width: 90%;
+    margin: auto;
+}
+
+.site-btn.btn-span {
+    border-radius: 10px;
+    border: 1px solid;
+    background-color: whitesmoke;
+    color: gray;
+}
+
+.slider-two .slick-track .product:hover span {
+    background-color: #93F1F0;
+}
+
+.site-slider-two .slider-btn .prev,
+.site-slider-two .slider-btn .next {
+    opacity: 0;
+    transition: opacity 1s ease;
+}
+
+.site-slider-two:hover .slider-btn .prev,
+.site-slider-two:hover .slider-btn .next {
+    opacity: 1;
+}
+
+.position-mid {
+    position: absolute;
+    top: 30%;
```

Comment:
if this is in the middle of something. Then this logic is wrong and will be rendered incorrect on different size screens.

Reply:
It's the unused code and has been deleted.